fastlane_core/lib/fastlane_core/tool_collector.rb

```diff
@@ -67,5 +106,32 @@ def did_show_message?
       File.write(path, '1') unless did_show
       did_show
     end
+
+    def determine_version(name)
+      begin
+        # We need to pre-load the version file because tools that are invoked through their actions
+        # will not yet have run their action, and thus will not yet have loaded the file which defines
+        # the module and constant we need.
+        require "#{name}/version"
+
+        # Go from :credentials_manager to 'CredentialsManager'
```

Comment:
Oh you choose it because of the `_` to demonstrate the upper casing of things. That's fine, let's keep it like this then.